codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializerCheckCompilerPluginComponent.scala

```diff
@@ -67,7 +67,7 @@ class SerializerCheckCompilerPluginComponent(
           .collect {
             case x: ImplDef => (x, x.symbol.annotations)
           }
-          .map(x => (x._1, x._2.filter(_.tpe.toString() == serializerType)))
+          .map(x => (x._1, x._2.filter(_.tpe.toString == serializerType)))
```

Comment:
Ditto, `x`, `_1`, `_2` &mdash; all to expand

Reply:
fixed